packages/discord.js/tsconfig.json

```diff
@@ -3,8 +3,12 @@
   "compilerOptions": {
```

Comment:
Could we enable could we enable [declarationMap](https://www.typescriptlang.org/tsconfig#declarationMap) too? It would be nice to be able to see the source code instead of the generated typings.